src/themes/default/stylesheets/main.scss

```diff
@@ -20,7 +20,7 @@ $size-desktop-large: 64em;
 // Variables used in this theme
 //////////////////////
 
-$font-family: 'Barlow','Open Sans',serif; // See Barlow import below...
+$font-family: 'Open Sans',serif; // See Barlow import below...
```

Comment:
We could use the system font stack here, e.g. 

```css
font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
```

From https://css-tricks.com/snippets/css/system-font-stack/

Also can remove the comment, no more barlow! Not sure if the google fonts import is still there, but that could be removed as well